src/OpenTelemetry/Trace/Export/Internal/MultiProcessor.cs

```diff
@@ -0,0 +1,60 @@
+﻿// <copyright file="MultiProcessor.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace OpenTelemetry.Trace.Export
+{
+    internal class MultiProcessor : SpanProcessor
+    {
+        private readonly IEnumerable<SpanProcessor> processors;
+
+        public MultiProcessor(IEnumerable<SpanProcessor> processors)
+        {
+            this.processors = processors ?? throw new ArgumentNullException(nameof(processors));
+        }
+
+        public override void OnEnd(Span span)
+        {
+            foreach (var processor in this.processors)
+            {
+                processor.OnEnd(span);
+            }
+        }
+
+        public override void OnStart(Span span)
+        {
+            foreach (var processor in this.processors)
+            {
+                processor.OnStart(span);
```

Comment:
do we need a try catch? or fail from the first failure? Perpahs it should be mentioned on `SpanProcessor`'s documentation for this method on what's the expected behavior for the implementation

Reply:
don't think so. Processors should never throw and if it throws span may catch it in `End`, catching something in MultiProcessor makes no sense - it's optional and probably never used for 90% of users.

Reply:
the reason to recover an issue here is so that one badly implemented processor will not affect others... troubleshooting errors would be much easier if at least some data in the Span will be delivered to the user...

May not be this PR.

Reply:
makes sense, will do this